.github/workflows/cypress.yml

```diff
@@ -41,6 +41,13 @@ jobs:
           ref: ${{ matrix.server-versions }}
           path: apps/viewer
 
+      - name: Checkout assistant
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          repository: nextcloud/assistant
+          ref: 'main'
+          path: apps/assistant
```

Comment:
Need to ensure that both assistant and testing app are also enabled in line 151 when setting up the Nextcloud instance to be used in the cypress tests